docs/guide/getting-started.md

```diff
@@ -26,11 +26,51 @@ if (typeof window.ethereum !== 'undefined') {
 
 You can review the full API for the `window.ethereum` object [here](./ethereum-provider.html).
 
+### Connecting to a test network.
+
+**Show test networks**<br />
+![image](https://user-images.githubusercontent.com/1038445/150907727-77fe4899-5238-4df6-8fa4-e6ae522e7d02.png)<br />
+
+![image](https://user-images.githubusercontent.com/1038445/150907769-c7ea63a8-6bd2-4f80-a376-a70243bb14b7.png)<br />
+
+**Connecting to Ropsten Test Network**<br />
+Use the network dropdown menu (which would normally be displaying Ethereum Mainnet like the image above) and select Ropsten Test Network<br />
+![image](https://user-images.githubusercontent.com/1038445/150908059-f86cd2eb-6d59-43e8-8fec-bacce425ebe0.png)<br />
+
+I recommend creating a test account just for the Ropsten Test Network. (Just give it a name)<br />
+
+![image](https://user-images.githubusercontent.com/1038445/150908294-22099c19-4b22-4587-939b-3f9f882b09e8.png)
+
+**Request 1 ETH Over the test network**<br />
+Navigate to https://faucet.metamask.io/
+Ensure you have the test account and test "Ropsten Test Network Selected"
+Click "reqest 1 ether from faucet"
+![image](https://user-images.githubusercontent.com/1038445/150909358-80beed6d-bebc-4602-90fb-183b15279228.png)
+
+A request will be sent to Metamask Client for you to sign. Click Next **WAIT Patiently for the transaction to be approved once Metamask has connected (may take a couple minutes)**
+![image](https://user-images.githubusercontent.com/1038445/150909613-6cb6d54e-3d43-4505-b8d6-7fa6b225e355.png)
+
+![image](https://user-images.githubusercontent.com/1038445/150909688-211f3398-e737-42db-bee2-e62d5e2ef5a8.png)
+
+![image](https://user-images.githubusercontent.com/1038445/150910232-703e1c26-4bdb-41aa-a174-7c10a9f3a6f8.png)
```

Comment:
I don't think the "User" section needs to be highlighted here. This shows your current balance but also allows you to donate to the faucet. Maybe it's worth showing a screenshot of the pending or confirmed transaction in the transaction list in MetaMask instead?